Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -202,8 +202,8 @@ private func pledgeCreditCardViewDataAndSelectedCard(
   return (data, backedCard)
 }
 
-private func showApplePayButton(for project: Project, applePayCapable: Bool) -> Bool {
-  return applePayCapable &&
+private func showApplePayButton(for project: Project, applePayDevice: Bool) -> Bool {
+  return applePayDevice &&
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kickstarter/ios-oss/blob/5b8f50da8451b55c1820ae8ec887b2a7e5064f91/Library/ViewModels/PledgePaymentMethodsViewModel.swift#L206"></span>Line should be 110 characters or less: currently 122 characters<br />
<code>line_length</code> <code>PledgePaymentMethodsViewModel.swift:206</code></td>
    </tr>
  </tbody>
</table>